src/cli/argument-parser.js

```diff
@@ -85,6 +85,49 @@ export default class CLIArgumentParser {
         return '\n' + wordWrap(DESCRIPTION, 2, getViewPortWidth(process.stdout));
     }
 
+    static _parseBrowserArg (browserArg) {
+        var currentBrowser = '';
+        var browsers       = [];
+
+        // NOTE: quotedChar contains ' or " - starting character of quotation when we are inside an enquoted part,
+        // or an empty string otherwise.
+        var quoteChar = '';
+
+        for (var i = 0; i < browserArg.length; i++) {
+            var currentChar = browserArg[i];
+
+            switch (currentChar) {
+            case ',':
+                if (quoteChar)
+                    currentBrowser += currentChar;
+                else {
+                    browsers.push(currentBrowser);
+                    currentBrowser = '';
+                }
+                break;
+
+            case '"':
+            case '\'':
+                if (quoteChar === currentChar)
+                    quoteChar = '';
+                else if (!quoteChar)
+                    quoteChar = currentChar;
+
+                currentBrowser += currentChar;
```

Comment:
My thought is that there is a a point of including quote char only if it is escaped by another type of quotes, e.g. `path:"/John's folder/firefox.exe"`. `'` is valid for file names even on Windows, `"` is valid for UNIX'es.